.deploy/nais/unleash-apitoken-preprod.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: unleash.nais.io/v1
```

Comment:
Disse burde vel være i en egen PR, med en eller annen funksjonalitet for å deploye de?

Reply:
I Ba-sak tror jeg vi bare deployet de ved bruk av `kubectl apply` i forkant. Om ikke annet må man i hvertfall huske å gjøre det. Ved egen PR kan det sikkert gjøres tilsvarende som i familie-tilbake:
https://github.com/navikt/familie-tilbake/blob/main/.github/workflows/deploy-unleash-api-token.yaml

Reply:
De er begge deployet m/ kubectl, men legger til separat workflow som tilbake og 👍 
https://github.com/navikt/familie-ks-sak/pull/475